xapian-letor/api/featuremanager_internal.cc

```diff
@@ -37,6 +37,7 @@
 using namespace Xapian;
 using namespace std;
 
+struct FileNotFound { };
```

Comment:
Note for future: I'm guessing we'll end up with a set of letor exceptions, with their own root, at which point this can move out to its own file and probably become a class. (Whether this inherits from the xapian-core root exception is something we'll have to discuss when we have some more experience working with the API.)